spring-support-root/spring-mvc-endpoint-support/src/main/java/org/divy/common/bo/endpoint/factory/DefaultHATEOASMVCEndpoint.java

```diff
@@ -15,25 +15,19 @@
 
 
     private final BOManager<E, UUID> manager;
-    private final AbstractAssociations<E> associations;
-    private final HyperMediaMapper<E, SpringMVCRepresentation> greetingHATEOSMapper;
+    private final HyperMediaMapper<E, SpringMVCRepresentation> hateosMapper;
     private final Class<E> type;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "type" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)